src/main/java/com/undertheriver/sgsg/memo/domain/Memo.java

```diff
@@ -79,4 +80,9 @@ public boolean equals(Object other) {
     public int hashCode() {
         return Long.hashCode(id);
     }
+
+
+    public void delete() {
+        setDeleted(true);
+    }
```

Comment:
```suggestion
    public void delete() {
        delete()
    }
```
객체를 set하는것보다 메시지를 주는형식이 더 객체지향적인 것 같습니다.
메서드가 어떤 것을 하는지 명확하게 구분할 수 있으며, 무분별하게 setter를 사용하게되면 코드파악이 어려워집니다